solvers/gams/mip/mip_to_qubo_reformulation.gms

```diff
@@ -0,0 +1,80 @@
+$setEnv GAMSINP "%INPUT%"
```

Comment:
mip_to_qubo_reformulation and qubo_solve should be named according to what they do. So creating gams representation of mip for the first and for the second gams to qubo formulation